### PR TITLE
[Release] 0.1.3

### PR DESCRIPTION
**Pull request by nickvander**
## Description of the change
- Created Apps Script to process Google docs, PDF, and Gmail threads. See `apps_script/README.md`.
- Added `third_party/g2docsmd-html` which is the base for converting files to markdown.
- Updated the main `README.md` for clarity.
- Created a `scripts/README.md` to better explain content processing.

## Motivation
Support additional sources of content such as PDF, Gmail and Google docs.

## Type of change
Choose one: (Feature request)

## Checklist

- [ X] I have performed a self-review of my code.
- [ X] I have added detailed comments to my code where applicable.
- [ X] I have verified that my change does not break existing code.
- [ X] My PR is based on the latest changes of the main branch (if unsure, please run `git pull --rebase upstream main`).
- [ X] I am familiar with the [Google Style Guide](https://google.github.io/styleguide/) for the language I have coded in.
- [ X] I have read through the [Contributing Guide](https://github.com/google/generative-ai-docs/blob/main/CONTRIBUTING.md) and signed the [Contributor License Agreement](https://cla.developers.google.com/about).

### CODE REVIEW

**Comment by kyolee415** [comment]
Great! The feature that everyone was asking for!!

File: demos/palm/python/docs-agent/apps_script/README.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+1. To save, click the "Save project" icon in the toolbar.
+1. Repeat the steps until all the `.gs` files are copied to your Apps Script project.
+1. Click the **+** icon next to **Services**.
+1. Scroll down and click **Drive API**.
+1. Click **Add**.
+
+You are now ready to edit the parameters on the `main.gs` file to select a folder
+in Google Drive and export emails from Gmail.
+
+![Apps Script project](../docs/images/apps-script-screenshot-01.png)
```
Comment (kyolee415) [inline demos/palm/python/docs-agent/apps_script/README.md:62]: This PNG image seems to be missing. But it's not a blocker for this pull request -- we can consider adding the image in the next pull request.

File: demos/palm/python/docs-agent/apps_script/README.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+The collection of scripts in this `apps_script` directory allows you to convert
+the contents of Google Drive folders and Gmail to Markdown files that are
+compatible with Docs Agent.
+
+The steps are:
+
+1. [Prepare a Google Drive folder](#1-prepare-a-google-driver-folder).
+2. [Mount Google Drive on your host machine](#2-mount-google-drive-on-your-host-machine).
+3. [Create an Apps Script project](#3-create-an-apps-script-project).
+4. [Edit and run main.gs on Apps Script](#4-edit-and-run-main-gs-on-apps-script).
```
Comment (kyolee415) [inline demos/palm/python/docs-agent/apps_script/README.md:12]: nit: `#4-edit-and-run-maings-on-apps-script`

File: demos/palm/python/docs-agent/apps_script/README.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+5. [Update config.yaml to include the mounted directory](#5-update-config-yaml-to-include-the-mounted-directory).
```
Comment (kyolee415) [inline demos/palm/python/docs-agent/apps_script/README.md:13]: nit: `#5-update-configyaml-to-include-the-mounted-directory`

**Review by kyolee415 — COMMENTED** [review]
I was able to verify that the main documented flow for setting up and launching Docs Agent was intact. Minor comments on the new README file. Thanks!

**Review by kyolee415 — COMMENTED** [review]
And I also verified that I was able to use the `.gs` files (which are included in this pull request) in my Apps Script project to convert Google Docs in my Google Drive folder into Markdown files. Great!

**Review by kyolee415 — APPROVED** [review]
Looks good to me!

File: demos/palm/python/docs-agent/apps_script/exportmd.gs
Comment (markmcd) [inline demos/palm/python/docs-agent/apps_script/exportmd.gs:1]: Sorry if I misled you here - this is 3p content (with local modifications, but still ultimately contains code sourced from outside of this package), so can you put the *patched* version into /third_party and then, say, symlink to that copy? The idea being that anything wholly first-party can go anywhere, but anything 3p, external or with mixed provenance should be tracked in /third_party.

It likely doesn't matter as this code is ultimately (c) google, but it is external to this package, so it'd be good to get the structure set up in a way you can copy for any future work.

Comment (nickvander) [reply demos/palm/python/docs-agent/apps_script/exportmd.gs:1]: Sorry about that. I think my latest patch takes care of this. Please take a look!

File: demos/palm/python/docs-agent/apps_script/exportmd.gs [hunk 4]
```diff
@@ -0,0 +1 @@
+/usr/local/google/home/nickvander/docs-agent-github3/third_party/docs-agent/g2docsmd-html/exportmd.gs
```
Comment (markmcd) [inline demos/palm/python/docs-agent/apps_script/exportmd.gs:1]: I think this needs to be a relative symlink so that it works for everyone.

**Review by markmcd — APPROVED** [review]

File: demos/palm/python/docs-agent/apps_script/exportmd.gs (hunk 4 above)
Comment (nickvander) [reply demos/palm/python/docs-agent/apps_script/exportmd.gs]: Thanks for catching that! Fixed

**Pull request merged**